pkg/taskservice/task_runner.go

```diff
@@ -312,7 +312,7 @@ func (r *taskRunner) fetch(ctx context.Context) {
 			}
 			tasks, err := r.doFetch()
 			if err != nil {
-				break
+				r.logger.Error("fetch task failed", zap.Error(err))
```

Comment:
Here, the break is a big deal.   You sure?

Reply:
This break here is actually useless. I added it back for clarity.